pkg/apis/alerting/v1/alerting.proto

```diff
@@ -575,4 +578,19 @@ message EphemeralDispatcherRequest {
 
 message EphemeralDispatcherResponse{
   TriggerAlertsRequest triggerAlertsRequest = 1;
+}
+
+message CachedState {
+  bool healthy = 1; 
+  bool firing = 2; 
```

Comment:
Are these mutually exclusive?

Reply:
In the backend when the internal evaluator uses these :

`firing = T = > healthy = F`
`healthy = T = > firing = F`
However the converse :
`healthy = F =/=> firing = T`

I'll add a comment on the proto to clarify this